### ISSUE TITLE
Exports are ignored

### ISSUE THREAD

**Issue by starlingjon** [lucasefe/eslint-plugin-align-assignments#23]
the following code is ignored:

```javascript
export const foo = 123;
export const fooBar = 321;
```

no solution is expected in this repo (it hasn't been modified for 6 years) but I figured it's worth opening the issue for completeness and possibly discussion.

it looks like #10 was opened for this but was closed, I tried that solution and got some false positives but after changing the line to the following it seems to work

```javascript
      // switching parent nodes (except when parent node is an export)
      if (node.parent !== previousNode.parent && node.parent.type !== 'ExportNamedDeclaration')
  ```
  
  this change is in `shouldStartNewGroup`
